demos/crossroad_camera_demo/crossroad_camera_demo.hpp

```diff
@@ -64,6 +64,9 @@ static const char no_show_processed_video[] = "Optional. No show processed video
 /// @brief message resizable input flag
 static const char input_resizable_message[] = "Optional. Enables resizable input with support of ROI crop & auto resize.";
 
+/// @brief Message list of monitors to show
+static const char utilization_monitors_message[] = "Optional. List of monitors to show.";
```

Comment:
```suggestion
static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
```

(presumably, you can still disable them using the keys)

Reply:
Updated.